### DISCUSSION TITLE
Download OMV for raspberry Pi 4 Download Error

### DISCUSSION THREAD

**Discussion by Larrtystech** [community/community#37055] [Other Feature Feedback, Questions, & Ideas]
I getting this error on my Pi when trying to download OMV.

wget -O- https://github.com/OpenMediaVault-Plugin-Developers/InstallScript/raw/master/Install | sudo bash
--2022-10-23 20:31:11--  https://github.com/OpenMediaVault-Plugin-Developers/InstallScript/raw/master/Install
Resolving github.com (github.com)... 140.82.114.4
Connecting to github.com (github.com)|140.82.114.4|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-23 20:31:12 ERROR 404: Not Found.

**Comment by FatChicken277** [comment]
The github link you are using does not exist, use this instead https://github.com/OpenMediaVault-Plugin-Developers/installScript/blob/master/install

**Comment by Larrtystech** [comment]
Thank you, Alejandro  I’m trying to download software called Open Media Vault while Raspberry Pi is running. Tried your script and received the following: wget -O- https://github.com/OpenMediaVault-Plugin-Developers/InstallScript/blob/master/Install--2022-10-23 21:09:02--  https://github.com/OpenMediaVault-Plugin-Developers/InstallScript/blob/master/InstallResolving github.com (github.com)... 140.82.112.3Connecting to github.com (github.com)|140.82.112.3|:443... connected.HTTP request sent, awaiting response... 404 Not Found2022-10-23 21:09:03 ERROR 404: Not Found.  I followed the URL and found a page of code   Sent from Mail for Windows From: Alejandro Ramírez C.Sent: Sunday, October 23, 2022 3:59 PMTo: community/communityCc: Larrtystech; AuthorSubject: Re: [community/community] Download OMV for raspberry Pi 4 Download Error (Discussion #37055) The github link dos is using does not exist, use this instead https://github.com/OpenMediaVault-Plugin-Developers/installScript/blob/master/install—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by FatChicken277** [comment]
MMM ok, so, try doing this:

1. Download installation script: `wget https://raw.githubusercontent.com/OpenMediaVault-Plugin-Developers/installScript/master/install -O omv-install.sh`
2. Add execution permission: `chmod +x omv-install.sh`
3. Execute: `./omv-install.sh`

And i give you an advice, avoid using commands without knowing what it does
